src/components/DBSettingsPage/index.jsx

```diff
@@ -170,6 +171,15 @@ class DBSettingsPage extends React.Component {
     this.setState({ uriChecked: true });
   }
 
+  uriPOnClick() {
+    const { databaseID } = this.props.match.params;
+    const dbInfo = this.getDatabaseInfo(databaseID);
+    navigator.clipboard.writeText(
+      `${`psql -h ${dbInfo.host} -p ${dbInfo.port} -d ${dbInfo.name} -u ${dbInfo.user}`}`
```

Comment:
@neelxie still missing a something the `-u` is supposed to be capital `-U` and the flag that prompts for a password `-W` is missing

Reply:
Unless this PR is strictly for copying issue and another for the actual command awaits

Reply:
@MutegekiHenry thank you, I have fixed that.

Reply:
Awesome!